test/cal_web/live/app_live_test.exs

```diff
@@ -24,6 +24,19 @@ defmodule CalWeb.AppLiveTest do
   end
 
 
+  test "changing date and posting new event", %{conn: conn} do
```

Comment:
@LuchoTurtle do you feel like you're learning more about calendars, dates and Google Calendar "internals" from this quest? 💭

Reply:
Cool. Hope you are able to capture some of the nuance in the dicaz 🤞🏼

P.S: Hope you aren’t working on a public holiday. 😜